db/migrate/20200821091915_add_gdpr_confirmation_check_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddGdprConfirmationCheckToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :privacy_confirmation, :boolean, default: false
```

Comment:
Since this is a boolean field, make attribute name more explicit, like 

`accepted_privacy_policy` so that method invocations will be cleaner